clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -86,7 +86,8 @@ func (s *schedulerLatencyMeasurement) Execute(config *measurement.MeasurementCon
 
 	var masterRegistered = false
 	for _, node := range nodes.Items {
-		if util.LegacyIsMasterNode(node.Name) {
+		node := node
```

Comment:
Let's remove it - it's unnecessary copy that isn't needed if the LegacyIsMasterNode isn't mutating the object (which it obviously shouldn't).

Reply:
ok